src/instabot.py

```diff
@@ -754,8 +756,9 @@ def explore_operation(self, operation, identifier):
             comment = False
             # comment
             if operation == 'comment':
-                comment = generate_comment(self.params['comments_list'])
-                self.console.log(comment + '\,')
+                commentString = generate_comment(self.params['comments_list'])
+                self.console.log('\n\n\tComment: "' + commentString + '" \n')
```

Comment:
Should we remove the last `\n` and add `\,` to have `Success` printed in the same line? Otherwise it looks like this:
```
[ 2018-03-27 16:11:55 ]  * Trying to comment "https://www.instagram.com/p/":  

	Comment: "Cool :star: :clap: Keep it up!" 

Success
```

Reply:
Other than that it works fine 👍 I'll merge once you reply to this ☝️ @eliareutlinger

Reply:
I didn't really knew where to should put that `Success` because of the long comments. But yes that'd look better.